pages/search-beta.tsx

```diff
@@ -122,8 +134,29 @@ const Search = ({ serverState, isServerRendered, serverUrl }: Props) => (
             {/* @ts-ignore  */}
             <Hits hitComponent={Hit} />
           </StyledHits>
+
           <FilterContainer>
```

Comment:
I  would have refactored the FilterContainer to its own file to avoid to much components with styles in this file

Reply:
This one when I tried moving to Filter component the whole CSS had turbulence . Can open it as separate task and then move all containers to respective components